aws/resource_aws_spot_instance_request.go

```diff
@@ -132,6 +132,11 @@ func resourceAwsSpotInstanceRequestCreate(d *schema.ResourceData, meta interface
 		spotOpts.LaunchGroup = aws.String(v.(string))
 	}
 
+	// Placement GroupName can only be specified when instanceInterruptionBehavior is not set to 'stop'
+	if v, exists := d.GetOkExists("instance_interruption_behavior"); v == "terminate" || !exists {
```

Comment:
I think this needs to be `v.(string) == ec2.InstanceInterruptionBehaviorTerminate`